portlock/portlock.py

```diff
@@ -0,0 +1,137 @@
+
+import errno
+import hashlib
+import socket
+import time
+
+port_n = 3
```

Comment:
https://www.python.org/dev/peps/pep-0008/#constants It is recommended by pep8 to use capital letters for constants.